.golangci.yml

```diff
@@ -3,37 +3,37 @@ issues:
   max-issues-per-linter: 0
   new-from-rev: origin/master
 linters:
-  presets: 
-  - bugs
-  - error
-  - format
-  - performance
-  - unused
+  presets:
+    - bugs
+    - error
+    - format
+    - performance
+    - unused
   disable:
-  - maligned
-  - scopelint
+    - maligned
+    - scopelint
+    - gomnd
```

Comment:
I fully support removing `gomnd` since I think it does more harm than good, but I also think any linter changes should be done in a separate PR so we can have a discussion about just doing that.

Reply:
We can discuss that in a seperate PR to remove gomnd